core/kazoo_number_manager/include/knm_phone_number.hrl

```diff
@@ -35,8 +35,7 @@
 -define(PVT_CREATED, <<"pvt_created">>).
 -define(PVT_TYPE, <<"pvt_type">>).
 
--define(KNM_DB_PREFIX_L, "numbers/").
--define(KNM_DB_PREFIX, <<?KNM_DB_PREFIX_L>>).
+-define(KNM_DB_PREFIX, "numbers/+").
```

Comment:
maybe add a `KNM_DB_PREFIX_encoded`and `KNM_DB_PREFIX_ENCODED`as you did for the MATCH_ACCOUNT stuff and replace in db_classification ? seems odd to see `KNM_DB_PREFIX` not applied in full in db_classification.
either revert the db_classification change or add the encoded/ENCODED ones.